src/App.js

```diff
@@ -60,90 +63,113 @@ function App() {
 
   return (
     <BrowserRouter>
-      <Header
-        currentUser={currentUser}
-        currentAvatar={currentAvatar}
-        changeUser={changeUser}
-        itemData={itemData}
-        baseUrl={baseUrl}
-      />
-      <Routes>
-        <Route path="/" element={<WelcomePage game={game} />} />
-        <Route
-          path="/game"
-          element={
-            <GamePage game={game} setGame={setGame} currentUser={currentUser} />
-          }
+      <ThemeProvider theme={theme}>
```

Comment:
So is this wrapped in everything and itll automatically format the theme for each page?